tasmota/support_command.ino

```diff
@@ -1860,7 +1860,11 @@ void CmndTeleperiod(void)
     if ((Settings.tele_period > 0) && (Settings.tele_period < 10)) Settings.tele_period = 10;   // Do not allow periods < 10 seconds
 //    TasmotaGlobal.tele_period = Settings.tele_period;
   }
-  TasmotaGlobal.tele_period = Settings.tele_period;        // Show teleperiod data also on empty command
+  TasmotaGlobal.tele_period = Settings.tele_period;        // Show teleperiod data also on empty command (Settings.tele_period>0)
+  if (Settings.tele_period = 0) {                          // Show teleperiod data also on empty command (Settings.tele_period=0)
```

Comment:
Boom!
Did you meant `if (Settings.tele_period == 0) {` -- classical C error.
That's why it's better to do it the opposite side: `if (0 == Settings.tele_period) {  ` in such case the compiler will complain that you try to assign a value to zero.